community/kernel/src/main/java/org/neo4j/tooling/GlobalGraphOperations.java

```diff
@@ -171,22 +201,54 @@ protected Relationship fetchNextOrNull()
      */
     public ResourceIterable<Label> getAllLabels()
     {
+        return getAllLabels( false );
+    }
+
+    /**
+     * Returns all labels currently in use in the underlying store. Labels are added to the store the first time
+     * they are used. This method guarantees that it will return all labels currently in use.
```

Comment:
_in use_ definition here is not very clear, corresponding `getAllLabels` comes with _historic_ example. Can we explain _in use_ also based on that one?

Also javadoc for relationship methods for now does not mention _in use_ anywhere